From eb1a50a20f875e0fafffdb487a437a9e28de343b Mon Sep 17 00:00:00 2001
From: Vladimir Lomov <vladimir@theor7.ktp>
Date: Tue, 2 Mar 2021 09:53:11 +0800
Subject: [PATCH 22/30] scripts: updated gen-work and methods

  - methods.py: use 'work.json' as json file with work instructions;
  - gen-work.py: write shuffled list into 'work.json' file and one with
    "direct" order into "work-do.json".

Signed-off-by: Vladimir Lomov <vladimir@theor7.ktp>
---
 scripts/gen-work.py | 4 ++--
 scripts/methods.py  | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/gen-work.py b/scripts/gen-work.py
index 2b75e6b..e5b3eee 100755
--- a/scripts/gen-work.py
+++ b/scripts/gen-work.py
@@ -28,7 +28,7 @@ for r in rngs:
 
 shfld = random.sample(work, k=len(work))
 
-with open(json_dir / "sample1.json", "w") as f:
+with open(json_dir / "work.json", "w") as f:
     json.dump({ "work" : shfld }, f, indent="  ")
-with open(json_dir / "sample2.json", "w") as f:
+with open(json_dir / "work-do.json", "w") as f:
     json.dump({ "work" : work }, f, indent="  ")
diff --git a/scripts/methods.py b/scripts/methods.py
index f8b930f..c57be80 100755
--- a/scripts/methods.py
+++ b/scripts/methods.py
@@ -116,7 +116,7 @@ if __name__ == '__main__':
 
     #  LOG_FILE_DIR = 'logs_0.1_0.3/'
     #  PARAMS_FILE = 'methods_0.1_0.3.json'
-    json_file = json_dir / "01_03.json"
+    json_file = json_dir / "work.json"
     if not os.path.isfile(json_file):
         print(f"Can't do this run, we need a '{json_file}' in '{json_dir}'!")
         sys.exit(2)
-- 
2.30.1

